### ISSUE TITLE
AirborneOperationalStatusV1Msg: supportsTargetChangeReport() is always false

### ISSUE THREAD

**Issue by mrksngl** [openskynetwork/java-adsb#31]
In [AirborneOperationalStatusV1Msg.java](https://github.com/openskynetwork/java-adsb/blob/e25a80a6fb172d46dc6a9f2a71bc6a2bea79c24a/src/main/java/org/opensky/libadsb/msgs/AirborneOperationalStatusV1Msg.java#L135) there is computation
```
byte target_change_report_capability = (byte) (capability_class_code & 0xC0);
return target_change_report_capability == 1 | target_change_report_capability == 2;
```
which is never true: the lowest two bits of `target_change_report_capability` will always be 0 after logical conjunction with `0xc0`.
Also, the disjunction in the return statement should be logical, i.e. `||`, rather than bitwise.

**Comment by fixje** [comment]
Thanks. There are two problems (i) missing bit shift, (ii) wrong operator as you already mentioned. Should rather be:

```
byte target_change_report_capability = (byte) ((capability_class_code & 0xC0) >>> 6);
return target_change_report_capability == 1 || target_change_report_capability == 2;
```

**Comment by fixje** [comment]
fixed in v3.2.0
